docs/astro.config.mjs

```diff
@@ -300,6 +300,11 @@ export default defineConfig({
               translations: { es: 'Dividir un pago entrante' },
               link: '/guides/split-payments/'
             },
+            {
+              label: 'One-time payment with debitAmount',
+              translations: { es: 'Pago único con debitAmount' },
+              link: '/guides/make-onetime-payment-debit/'
```

Comment:
Ok, I have no problems with this file name, but I'm also trying to think ahead. What if there's another guide we want to do that's also for one time payments using a debit amount, just not using the remittance use case?